lib/opsgenie_alert_tagging.rb

```diff
@@ -21,4 +21,67 @@ def list_all_alerts(date)
 
     results["data"]
   end
+
+  def alerts_which_need_updating
+    response = list_all_alerts(date)
+    results = []
+    response.each do |raw_alert|
```

Comment:
Do you have an example of what you were thinking for this?
The only way we can see map working is by nesting i.e `response.map { |raw_alert| raw_alert.map { |k,v| [k, "inhours"] }.to_h } `, which doesn't seem as clear as before.

Reply:
Currently we have an array `results` that we add to at the end of the `each`, this is a classic case for a `map` instead. Your code becomes `results = response.map` instead of `results = []; response.each`.